### ISSUE TITLE
Request: move window to another display with keyboard

### ISSUE THREAD

**Issue by kranix0** [newlooper/VirtualSpace#60]
I'm seriously impressed with the development of this application!

I see it's possible to have rules that move a window to a different screen/display. It would also be great if I could use keyboard shortcuts to move the selected window to a display positioned up/down/left/right of where it currently resides.

So for example, I could press Ctrl+Alt+Shift+Right to move a window to the monitor positioned to the right.

**Comment by akoyich** [comment]
Yes! Definitely need to be able to move windows left/right/up/down and follow that window.
